src/main/java/course_project/models/Cell.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
```

Comment:
Opt: what is this?)

Reply:
Started to write at the university, maybe it is some of their IDE tags :-)